lib/process-services-cloud/src/lib/process/start-process/services/start-process-cloud.service.ts

```diff
@@ -96,7 +96,9 @@ export class StartProcessCloudService extends BaseCloudService {
         const url = `${this.getBasePath(appName)}/rb/v1/process-instances`;
         payload.payloadType = 'StartProcessPayload';
 
-        return this.post(url, payload);
+        return this.post(url, payload).pipe(
+            map((result: any) => result.entry)
```

Comment:
This might be potentially problematic on apps, as maybe someone is using it and it doesn't get the entry or already maps outside for entry.

Reply:
As the fact that this would change in a more invasive way i would say that's ok